from evodoc.exception.evoDocException import EvoDocException


class DbException(EvoDocException):
    """
    This exception is raised in entity module
    """

    # def __init__(self, errorCode, message, invalid=[]):
    #     self.errorCode = errorCode
    #     self.message = message
    #     self.invalid = invalid
